web/modules/custom/bnf/graphql/bnf_extension.extension.graphqls

```diff
@@ -40,3 +40,15 @@ extend type NodeGoPage {
 extend type Link {
   id: String
 }
+
+extend type ParagraphNavGridManual {
```

Comment:
Rather than extending each paragraph type that has a reference field, extend the underlying field type.

See
https://github.com/danskernesdigitalebibliotek/dpl-cms/blob/f5c18fe98346501bc4459e2de05e4e61fd9970c8/web/modules/custom/bnf/graphql/bnf_extension.extension.graphqls#L30

and

https://github.com/danskernesdigitalebibliotek/dpl-cms/pull/2402/commits/adba1b6b717a0d0c8a0504a8eb2913f209b3b8fa

Reply:
But, the underlying fieldtype more or less means the same thing - or am i misunderstading something?

<img width="725" alt="Screenshot 2025-07-04 at 13 46 21" src="https://github.com/user-attachments/assets/fed45651-3c80-46b7-96b6-40bb4ec58ab4" />